topic-controller/resources/openshift-template.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: strimzi-topic-controller
+  labels:
+    app: strimzi
+---
+apiVersion: v1
+kind: ClusterRole
+metadata:
+  name: strimzi-topic-controller-role
+  labels:
+    app: strimzi
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - patch
+  - update
+---
+apiVersion: v1
+kind: RoleBinding
+metadata:
+  name: strimzi-topic-controller-binding
+  labels:
+    app: strimzi
+subjects:
+  - kind: ServiceAccount
+    name: strimzi-topic-controller
+roleRef:
+  kind: ClusterRole
+  name: strimzi-topic-controller-role
+apiGroup: v1
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: strimzi-topic-controller
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: strimzi-topic-controller
+    spec:
+      serviceAccountName: strimzi-topic-controller
+      containers:
+        - name: strimzi-topic-controller
+          image: strimzi/topic-controller:latest
+          env:
+            - name: TC_CM_LABELS
+              value: "strimzi.io/kind=topic"
+            - name: TC_ZK_SESSION_TIMEOUT
+              value: "20seconds"
+            - name: TC_PERIODIC_INTERVAL
+              value: "15minutes"
```

Comment:
Now I got more information about my previous comment on these values. Does this kind of configuration "seconds" or "minutes" come from some library (i.e. ZK client) are you using ? Why not using the second as unit of measure and getting rid of "seconds", "minutes" and so on ?

Reply:
Per previous comment, it originates with the names of the elements of `TimeUnit`. We could support abbreviations (`10s`) too/instead. I agree it would be better, but I'll do that in a later PR.